.github/test.md

```diff
@@ -0,0 +1 @@
+"This file should be ignored by git"
```

Comment:
- [ ] Kindly remove the ` test.md` file from the` .github` folder and add it to the root directory. The `test.md ` file should also be ignored by `git` which means you are to add it to` .gitignore` file as well just as you did for the `node modules`.

The` .github `folder is only supposed to contain the workflow linters. 😄 Kindly note that 👍🏽